app/cmd/cli/root.go

```diff
@@ -78,7 +80,7 @@ var startCmd = &cobra.Command{
 		if testnet {
 			genesisType = app.TestnetGenesisType
 		}
-		tmNode := app.InitApp(datadir, tmNode, persistentPeers, seeds, remoteCLIURL, keybase, genesisType)
+		tmNode := app.InitApp(datadir, tmNode, persistentPeers, seeds, remoteCLIURL, keybase, genesisType, cacheSize)
```

Comment:
This cache-size doesn't merit a flag.

Also, this is just ctxCache... Let's be specific here with naming

Reply:
After reviewing the entire code, I realized you never made a config option for this in the file. Please do that instead of the flag